specification/resources/resource-manager/Microsoft.Authorization/stable/2020-05-01/privateLinks.json

```diff
@@ -56,7 +56,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/PrivateLinkAssociationProperties"
+              "$ref": "#/definitions/PrivateLinkAssociationObject"
```

Comment:
Instead of introducing this new model, how about we just reuse the `PrivateLinkAssociation` here?
`PrivateLinkAssociation` has some more properties comparing with this `PrivateLinkAssociationObject`, but those are read-only and we do not have to provide them in the request.
In the SDK, usually it is quite common that the body parameter is the same as the response.

Reply:
`PrivateLinkAssociation` uses `PrivateLinkAssociationPropertiesExpanded` which contains more properties than is needed for the request, but is needed for the response